rwdclient.py

```diff
@@ -0,0 +1,80 @@
+import sys
+import os
+import unittest
+import xmlrunner
+import inspect
+from module.cliparser import parse
+from module.buildcap import build_capabilities
+from selenium import webdriver
+
+# exit codes
+SCRIPT_ERROR = 3
+TEST_UNSTABLE = 2
+TEST_FAILED = 1
+TEST_PASSED = 0
+ret = TEST_PASSED
+
+args = parse()
+
+desired_capabilities = build_capabilities(args)
+
+if (args.test_script and not(os.path.exists(args.test_script))):
+    print "Error, file %s does not exist" % (args.test_script)
+    sys.exit(SCRIPT_ERROR);
```

Comment:
We probably need another error code for this. Something like "SYSTEM_ERROR", as if the script is not there, it's not an error in the user script, it's a problem in our system.